Channels/Production/release_notes.md

```diff
@@ -1,24 +1,39 @@
-## September 27 2022 Production Release ([v1.1.2089.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.2089.01))
```

Comment:
@asksep, this is identical to the Insider notes, other than saying "Production" instead of "Insider"